marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mastering/updateOptions.api

```diff
@@ -0,0 +1,14 @@
+{
+    "functionName": "updateOptions",
```

Comment:
I am wondering if we need to distinguish between matching and merging in this function. Also, @ryanjdew - while we prefer matching/merging steps to mastering steps, is it reasonable to still refer to "mastering" as a subsystem, such that it makes sense to have a MasteringService as opposed to separate MatchingService and MergingService interfaces? (I kind of prefer just saying "mastering")

Reply:
Yes, I will rename this to updateMergeOptions.